payments/views/webhooks.py

```
@@ -69,37 +84,13 @@ def post(self, request):
 
         return Response({"status": "ok"})
 
-    def validate_signature(self, request):
-        """
-        Validate the signature of the request and return the parsed event.
+    def _determine_gateway_type(self, request):
```

Ruff (0.14.8)

Unused method argument: `request`

(ARG002)